content/en/blog/2019/dns-cert/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Istio DNS certificate management
+description: Provision and manage DNS certificates in Istio.
+publishdate: 2019-11-08
+attribution: Lei Tang (Google)
+keywords: [security, kubernetes, certificates, DNS]
+target_release: 1.4
+---
+
+By default, Citadel manages the DNS certificates of the Istio control plane.
+Citadel is a large component that maintains its own private signing key, and acts as a Certificate Authority (CA).
+
+New in Istio 1.4, we introduce a feature to securely provision and manage DNS certificates
```

Comment:
should we state it is experimental feature here?

Reply:
The user guide has stated it is an experimental feature.